projects/storefrontlib/layout/main/storefront.component.ts

```diff
@@ -26,6 +26,7 @@ import { StorefrontOutlets } from './storefront-outlets.model';
 @Component({
   selector: 'cx-storefront',
   templateUrl: './storefront.component.html',
+  host: { ngSkipHydration: 'true' },
```

Comment:
This disables all hydration. Useful to know as a way to disable it from the root but lets remove it.